x-pack/plugins/cases/server/plugin.ts

```diff
@@ -59,11 +59,13 @@ export interface PluginStartContract {
 
 export class CasePlugin {
   private readonly log: Logger;
+  private readonly kibanaVersion: PluginInitializerContext['env']['packageInfo']['version'];
   private clientFactory: CasesClientFactory;
   private securityPluginSetup?: SecurityPluginSetup;
   private lensEmbeddableFactory?: LensServerPluginSetup['lensEmbeddableFactory'];
 
   constructor(private readonly initializerContext: PluginInitializerContext) {
+    this.kibanaVersion = initializerContext.env.packageInfo.version;
```

Comment:
@jonathan-buttner I asked the core team and this is the only way we can get the current kibana version. I passed it to the cases' client through the client factory. Do you think there is a better way of passing this information to the routes?